src/Microsoft.AspNet.Mvc.Xml/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+    "description": "Contains DataContractSerializer and XmlSerializer based input and output formatters",
+    "version": "6.0.0-*",
+    "compilationOptions": {
+        "warningsAsErrors": false
```

Comment:
I think we should have true by default .. .did you copy from one of the razor projects?